tool/tctl/common/resource_command.go

```diff
@@ -986,6 +1122,20 @@ func (rc *ResourceCommand) Delete(ctx context.Context, client *authclient.Client
 	if resourceHandler, found := resources.Handlers()[rc.ref.Kind]; found {
 		if err := resourceHandler.Delete(ctx, client, rc.ref); err != nil {
 			if trace.IsNotImplemented(err) {
+				// Connectors is a special case. As it's the only meta-resource we have,
+				// it's easier to special-case it here instead of adding a case in the
+				// generic [resources.Handler].
+				if rc.ref.Kind == types.KindConnectors {
+					return trace.BadParameter(
+						"%q type is ambiguous. To delete the resource, please use its exact type. Possible resource identifiers are: %s",
+						types.KindConnectors,
+						[]string{
+							types.KindGithubConnector + "/" + rc.ref.Name,
+							types.KindOIDCConnector + "/" + rc.ref.Name,
+							types.KindSAMLConnector + "/" + rc.ref.Name,
+						},
+					)
+				}
```

Comment:
I'd say we can even move it to before the `if resourceHandler, found := resources.Handlers()[rc.ref.Kind]; found {` line